README.md

```diff
@@ -50,20 +50,23 @@ conda activate pudl-archiver
 A CLI is provided for creating and updating archives. The basic usage looks like:
 
 ```
-pudl_archiver {list_of_datasources}
+pudl_archiver --datasets {list_of_datasources}
 ```
 
 This command will download the latest available data and create archives for each
 requested datasource requested. The supported datasources include `eia860`, `eia861`, `eia923`,
 `eia_bulk_elec`, `epacems`, `epacamd_eia`, `ferc1`, `ferc2`, `ferc6`, `ferc60`,
 `ferc714`, `eia860m`.
 
-There are also two optional flags available, `--sandbox` and `--initialize`. The
-sandbox flag is used for testing. It will only interact with Zenodo's
-[sandbox](https://sandbox.zenodo.org/) instance. The initialize flag is used when
-creating an archive for a new dataset that doesn't currently exist on zenodo.
-If successful, this command will automatically add the new Zenodo DOI to the
-`dataset_doi.yaml` file.
+There are also four optional flags available:
+* `--sandbox`: used for testing. It will only interact with Zenodo's
+  [sandbox](https://sandbox.zenodo.org/) instance.
+* `--initialize`: used for creating an archive for a new dataset that doesn't
+  currently exist on zenodo. If successful, this command will automatically add
+  the new Zenodo DOI to the `dataset_doi.yaml` file.
+* `--dry-run`: used for testing, it ignores all Zenodo write operations.
+* `--all`: shortcut for archiving all datasets that we have defined archivers
```

Comment:
Do the GitHub runners have enough disk to do all of the datasets at once? I think they only have 14GB total, some of which will be used for the OS and software packages.

I guess we could easily break the different datasets out into different scheduled processes so they don't all download everything at the same time.

Reply:
Or would the `job.strategy` kick off totally independent runners? In that case I guess we just break them down so each job fits within the given resources of the runner. This will be great!

Reply:
`job.strategy` kicks off totally different runners!
In theory we could have some clever caching so we only have to spend the runner-minutes on building the env once. But seeing as we're living large on the open-source free minutes that seems kind of low on the list.

Reply:
That being said - this `--all` functionality turned out to not be immediately useful - I'm happy to rip it out.